src/goose/brokers/sqs/broker.clj

```diff
@@ -0,0 +1,82 @@
+(ns goose.brokers.sqs.broker
+  (:require
+   [goose.broker :as b]
+   [goose.brokers.sqs.worker :as sqs-worker]
+   [goose.brokers.sqs.sqs-requests :as sqs-request]
+   [cognitect.aws.client.api :as aws]
+   [goose.utils :as u]))
+
+(defprotocol Close
+  "Closes connection to SQS Message Broker."
+  (close [this]))
+
+(defrecord SQS [client opts]
+  b/Broker
+  ;; SQS supports nothing more than:
+  ;;   - Enqueue a job
+  ;;   - Start a worker process
+  ;;   - Enqueued jobs API
+  ;;   - Batch jobs (Will implement later)
+
+  (enqueue
+    [this job]
+    (let [{:keys [client]} this
+          {:keys [queue-url]} opts]
+
+      (sqs-request/enqueue client queue-url (u/encode-to-str job))))
+
+  (start-worker
```

Comment:
For background processing, clients might want to write to different queues, and a worker could process jobs off different queues. Need to design the enqueue & worker API keeping this in mind.

Reply:
`different queues` as in you mean different SQS services(in same account) and different queues within each service. Or you just mean we will allow Only one AWS account with a SQS service where they can connect to multiple queues?

Or you mean support for different accounts of AWS also?